### DISCUSSION TITLE
How to use alertmanager for docker install

### DISCUSSION THREAD

**Discussion by WayneShen2** [NetApp/harvest#3173] [Q&A]
I would like use alertmanager, how to Integrate in docker install

**Comment by rahulguptajss** [comment]
@WayneShen2 You may try steps listed [here](https://medium.com/javarevisited/monitoring-setup-with-docker-compose-part-3-alertmanager-5d0a2d4a5612) for poc.

**Comment by WayneShen2** [comment]
Thank you.
